rocAL/include/meta_data/meta_data.h

```diff
@@ -48,6 +48,10 @@ typedef struct BoundingBoxCord_ {
 
 typedef std::vector<BoundingBoxCord> BoundingBoxCords;
 typedef std::vector<int> Labels;
+
+typedef std::vector<uint8_t> AsciiComponent;
```

Comment:
please remove AsciiComponent and use uint8_t * instead

Reply:
Made changes to use shared_ptr